django_yubin/management/commands/send_test_mail.py

```diff
@@ -0,0 +1,58 @@
+# encoding: utf-8
+from __future__ import absolute_import, unicode_literals
+
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from ...messages import BasicHTMLEmailMessageView
+
+
+class Command(BaseCommand):
+    help = 'Send a simple email in order to check connection parameters.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-s',
+            '--subject',
+            dest='subject',
+            default='Hello World',
+            help='Email subject.',
+        )
+
+        parser.add_argument(
+            '-c',
+            '--content',
+            dest='content',
+            default='Hello World',
+            help='Email content.',
+        )
+
+        parser.add_argument(
+            '-t',
+            '--to',
+            dest='to',
+            default='',
+            help='Email address to send message.',
+        )
+
+    def handle(self, *args, **options):
+        from_email = settings.DEFAULT_FROM_EMAIL
+        to = options['to']
+        if to:
+            to_list = [to]
+        else:
+            to_list = [x[1] for x in settings.ADMINS if len(x) > 0]
+
+        if not to_list:
+            sys.stdout.write('Please provide an email address or set a valid settings.ADMINS configuration\n')
```

Comment:
Thanks for the PR @morenosan.

Since we are in a Django command, I think it would be better to write error messages using `self.stderr.write` or raising a `CommandError`.

Reply:
You're right. Raising a CommandError exception is a better option.